### ISSUE TITLE
Screenreader/ browser compatibility for all combinations

### ISSUE THREAD

**Issue by MaryS94** [RITAccess/blockly#22]
Currently supported browser screen reader combinations.

|                              | **Chrome**| **Firefox**       | **Internet Explorer**| **Safari**
-------------------------|---------------|-------------------|--------------------------|--------------
**Voiceover**          |✔Supported|**X** Incompatible|        N/A                   |✔Supported
**NVDA**                |✔Supported|✔Supported          |Untested                  |      N/A
**JAWS**                |**X** Incompatible |✔Supported|Untested                  |      N/A

**Comment by MaryS94** [comment]
Must press NVDA key & Space  to use Blockly Hotkeys. This should not be a problem for users because the same keyboard shortcut is needed to insert text on a webpage.

**Comment by MaryS94** [comment]
**Firefox is working on known live regions issue with voiceover which is why it is incompatible**   
[see this link for details](https://bugzilla.mozilla.org/show_bug.cgi?id=1198336)

**Comment by MaryS94** [comment]
Safari now works using "aria-label" on the blockReader div and adding the role "alert" to it. This same method does NOT work with chrome which changes in inner text of the div.  The code for chrome and safari are clearly distinguished so it's not confusing.
